### ISSUE TITLE
This problem wont let me train i tried

### ISSUE THREAD

**Issue by CYBORGX002** [iperov/DeepFaceLab#5446]
this keeps happening after i installed my GTX1660 non super card please help me to solve it 

[new] No saved models found. Enter a name of a new model : ju
ju

Model first run.

Choose one or several GPU idxs (separated by comma).

[CPU] : CPU
  [0] : NVIDIA GeForce GTX 1660

[0] Which GPU indexes to choose? :
0
[0] Autobackup every N hour ( 0..24 ?:help ) :
0
[n] Write preview history ( y/n ?:help ) :
n
[0] Target iteration :
0
[n] Flip SRC faces randomly ( y/n ?:help ) :
n
[y] Flip DST faces randomly ( y/n ?:help ) :
y
[8] Batch_size ( ?:help ) :
8
[128] Resolution ( 64-640 ?:help ) :
128
[f] Face type ( h/mf/f/wf/head ?:help ) :
f
[liae-ud] AE architecture ( ?:help ) :
liae-ud
[256] AutoEncoder dimensions ( 32-1024 ?:help ) :
256
[64] Encoder dimensions ( 16-256 ?:help ) :
64
[64] Decoder dimensions ( 16-256 ?:help ) :
64
[22] Decoder mask dimensions ( 16-256 ?:help ) :
22
[n] Eyes and mouth priority ( y/n ?:help ) :
n
[n] Uniform yaw distribution of samples ( y/n ?:help ) :
n
[n] Blur out mask ( y/n ?:help ) :
n
[y] Place models and optimizer on GPU ( y/n ?:help ) :
y
[y] Use AdaBelief optimizer? ( y/n ?:help ) :
y
[n] Use learning rate dropout ( n/y/cpu ?:help ) :
n
[y] Enable random warp of samples ( y/n ?:help ) :
y
[0.0] GAN power ( 0.0 .. 5.0 ?:help ) :
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) :
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) :
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
none
[n] Enable gradient clipping ( y/n ?:help ) :
n
[n] Enable pretraining mode ( y/n ?:help ) :
n
Initializing models: 100%|###############################################################| 5/5 [00:02<00:00,  1.87it/s]
Loading samples: 100%|############################################################| 2385/2385 [00:06<00:00, 365.63it/s]
Loading samples: 100%|############################################################| 3964/3964 [00:10<00:00, 376.61it/s]
Press any key to continue . . .

![Capture](https://user-images.githubusercontent.com/95993897/145815978-97d93b7d-0656-41fd-9b23-25f527368101.PNG)

**Comment by qkum** [comment]
Are you joking?

Try google translate "Press any key to continue..."

**Comment by CYBORGX002** [comment]
> Are you joking?
> 
> Try google translate "Press any key to continue..."

that closes the console im not joking

**Comment by CYBORGX002** [comment]
> Are you joking?
> 
> Try google translate "Press any key to continue..."

i tried soo mny times training wont start it says press any key to continue then when i press any key it quits

**Comment by Lazadum** [comment]
> Are you joking?
> 
> Try google translate "Press any key to continue..."

Why do you even answer when you're clearly too stupid to help?

**Comment by Lazadum** [comment]
> > Are you joking?
> > Try google translate "Press any key to continue..."
> 
> i tried soo mny times training wont start it says press any key to continue then when i press any key it quits

I have the exact same problem but instead of quitting, nothing happens after pressing the key.
It's really weird because the training worked 2 or 3 times fine and I didnt change any settings

**Comment by CYBORGX002** [comment]
> 

that's really weird

**Comment by CYBORGX002** [comment]
> > > Are you joking?
> > > Try google translate "Press any key to continue..."
> > 
> > 
> > i tried soo mny times training wont start it says press any key to continue then when i press any key it quits
> 
> I have the exact same problem but instead of quitting, nothing happens after pressing the key. It's really weird because the training worked 2 or 3 times fine and I didnt change any settings

that's really weird

**Comment by Evil0Dead** [comment]
@CYBORGX002  r u running on windows 11?

**Comment by KaelumKrispr** [comment]
Exactly the same thing happens to me, I've followed a few tutorial and mine keeps getting stuck where you are, I'm on a 3080ti with the right version installed

**Comment by CYBORGX002** [comment]
> @CYBORGX002 r u running on windows 11?

nah bro im on w10 pro

**Comment by CYBORGX002** [comment]
> Exactly the same thing happens to me, I've followed a few tutorial and mine keeps getting stuck where you are, I'm on a 3080ti with the right version installed

damn I installed my old 1650 its working fine bt 1660 didn't even work

**Comment by Evil0Dead** [comment]
there's a work around way but it makes training takes forever.. you can try train as quick .. I know it would fail 3-4 times (press any key to continue) but it will work on quick train . let me know if this works for you.

BTW .. this will works on CPU you may see cpu usage is 100% .. till it get fixed.

**Comment by Lazadum** [comment]
I know now what 

> > > > Are you joking?
> > > > Try google translate "Press any key to continue..."
> > > 
> > > 
> > > i tried soo mny times training wont start it says press any key to continue then when i press any key it quits
> > 
> > 
> > I have the exact same problem but instead of quitting, nothing happens after pressing the key. It's really weird because the training worked 2 or 3 times fine and I didnt change any settings
> 
> that's really weird



I know now what caused the issue for me. When it asked me to place model and optimizer and GPU and changed it to no instead of yes and its working now, idk why tho.

**Comment by CYBORGX002** [comment]
> 
I have found something weird I uninstalled my GPU driver and it started working but when I re-installed it ain't. and the thing u mentioned I tried that and it started working on CPU instead

**Comment by CYBORGX002** [comment]
> I know now what
> 
> > > > > Are you joking?
> > > > > Try google translate "Press any key to continue..."
> > > > 
> > > > 
> > > > i tried soo mny times training wont start it says press any key to continue then when i press any key it quits
> > > 
> > > 
> > > I have the exact same problem but instead of quitting, nothing happens after pressing the key. It's really weird because the training worked 2 or 3 times fine and I didnt change any settings
> > 
> > 
> > that's really weird
> 
> I know now what caused the issue for me. When it asked me to place model and optimizer on GPU I changed it to 'no' instead of 'yes' and its working now, idk why tho. (I have a GTX 1070 ti)

I have found something weird I uninstalled my GPU driver and it started working but when I re-installed it ain't. and the thing u mentioned I tried that and it started working on CPU instead

**Comment by balakius** [comment]
I think the problem is caused by the new nvidia drivers, i have the same problem since the last december nvidia driver update. If you rollback to november drivers it must work fine.

**Comment by chiragp007** [comment]
@CYBORGX002  did you solve the problem?

I am using "train Quick96"(from DeepFaceLab_DirectX12). After initializing the model and loading samples, it does not start Training (if I use Nvidia GTX1660Ti). However, it works when I go with Intel UHD graphics.

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
